package exception;

public class AR {

	public static void main(String[] args) {
		
		int money=200;
		int moneyspend=0;
		
		// TODO Auto-generated method stub
		try{  
			
			
		int 	to7tal=money/moneyspend;
		System.out.print(total);
		    //  int data=50/0;  
		   }catch(ArithmeticException e)
		{
			   System.out.println(e);
	    }  
		   System.out.println("rest of the code...");  
	}

	}


